astropy/coordinates/solar_system.py

```diff
@@ -0,0 +1,302 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+This module contains convenience functions for retrieving solar system
+ephemerides from jplephem.
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import numpy as np
+from ..extern import six
+from .sky_coordinate import SkyCoord
+from ..utils.data import download_file
+from .. import units as u
+from ..constants import c as speed_of_light
+from .representation import CartesianRepresentation
+from .builtin_frames import GCRS, ICRS
+from .builtin_frames.utils import get_jd12, cartrepr_from_matmul
+from .. import _erfa
+
+__all__ = ["get_body", "get_moon", "get_barycentric_body_position"]
+
+KERNEL = None
+
+"""
```

Comment:
Not completely sure, but do the docstrings get picked up for the documentation if they are put before the variable, or does it have to be after?

Independently: should this be exported in `__all__`? (e.g., `time.core` exports `TIME_SCALES`). If so, it probably should be renamed to `SOLAR_SYSTEM_BODIES`.